docs/topics/loaders.rst

```diff
@@ -35,6 +35,12 @@ Then, you start collecting values into the Item Loader, typically using
 the same item field; the Item Loader will know how to "join" those values later
 using a proper processing function.
 
+.. note:: Collected data is internally stored as lists,
+   allowing to add several values to the same field.
+   If an ``item`` argument is passed when creating a loader,
+   each of the item's values will be stored as-is if it's already
```

Comment:
stored as-is → converted into a list?

Reply:
Not necessarily, since this piece of code uses [`arg_to_iter`](https://github.com/scrapy/scrapy/blob/1.7.4/scrapy/utils/misc.py#L19-L30), non-list iterables are not converted.

Reply:
Then “stored as lists” is not correct, and they are stored as iterables? Won’t there be issues if a generator is used, when trying to add more values to it?

Reply:
True, although that's what all of the `add_*` methods do, because that's what `arg_to_iter` does. _Generally_, this isn't a problem because items and dicts are wrapped in lists, and the Selector query methods return SelectorLists.
Not sure I like it either, but I think making such change is probably out of the scope of this PR. Perhaps this grants a separate discussion/patch.